docs/templates.md

```diff
@@ -254,6 +254,13 @@ The following methods are defined. See also the `List` type.
 
 * `.join(separator: Template) -> Template`
 
+### List<Trailer> type
```

Comment:
nit: move before `ListTemplate` because `List<Trailer>` is a `List<_>` type, and this section only lists additional methods.